weed/server/filer_grpc_server_batch_lookup_test.go

```
@@ -0,0 +1,393 @@
+	if _, err := lookup(context.Background(), []string{"9"}); err != nil {
+		t.Fatal(err)
+	}
+	if cachedCalls != 1 || authoritativeCalls != 1 {
+		t.Fatalf("ordinary lookup did not keep cached path: cached=%d authoritative=%d", cachedCalls, authoritativeCalls)
+	}
+}
+
+func repeatedBatchLookupRequest(count int, request *filer_pb.LookupDirectoryEntryRequest) *filer_pb.LookupDirectoryEntriesRequest {
+	requests := make([]*filer_pb.LookupDirectoryEntryRequest, count)
+	for index := range requests {
+		copy := *request
```

golangci-lint (2.13.2)

[error] copylocks: assignment copies lock value to copy: github.com/seaweedfs/seaweedfs/weed/pb/filer_pb.LookupDirectoryEntryRequest contains google.golang.org/protobuf/runtime/protoimpl.MessageState contains sync.Mutex

(govet)